python/manylinux1/scripts/build_thrift.sh

```diff
@@ -17,10 +17,7 @@
 # under the License.
 
 export THRIFT_VERSION=0.12.0
-THRIFT_DOWNLOAD_PATH=thrift/${THRIFT_VERSION}/thrift-${THRIFT_VERSION}.tar.gz
-curl -sL \
-  "https://www.apache.org/dyn/mirrors/mirrors.cgi?action=download&filename=${THRIFT_DOWNLOAD_PATH}" \
-  -o thrift-${THRIFT_VERSION}.tar.gz
+wget https://archive.apache.org/dist/thrift/${THRIFT_VERSION}/thrift-${THRIFT_VERSION}.tar.gz
```

Comment:
The other manylinux images are alredy using the dist url. We rarely re-build this image so I think it's fine to keep as is.